### PR TITLE
Tagging ingested IOC (HTTPFeedApiModule bug)

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/35301

## Description
Fixed an issue where feeds were not tagging ingested IOCs (HTTPFeedApiModule bug)

## Screenshots

After fix:
image

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ApiModules/ReleaseNotes/2_2_0.md [hunk 1]
```diff
@@ -3,3 +3,6 @@
 ##### JSONFeedApiModule
 - Added support for grid fields hierarchy.
 - Added option for flat JSON response.
+
+##### HTTPFeedApiModule
```
Comment (yaakovi) [inline Packs/ApiModules/ReleaseNotes/2_2_0.md:7]: revert this change, there shouldn't be release notes for API modules

File: Packs/ApiModules/ReleaseNotes/2_2_0.md [hunk 2]
```diff
@@ -3,3 +3,6 @@
...
+- Fixed an issue where tags field was not shown.
```
Comment (Bargenish) [inline Packs/ApiModules/ReleaseNotes/2_2_0.md:8]: Let's create a new release note for this pack and bump the version.

**Review by yaakovi — CHANGES_REQUESTED** [review]
move the release notes to all the relevant feeds

**Review by Bargenish — CHANGES_REQUESTED** [review]
Add RN for all the packs that using this API module.
```
Packs/FeedPlainText 
Packs/FeedFeodoTracker 
Packs/FeedMalwareDomainList
Packs/FeedBlocklist_de
Packs/FeedBruteForceBlocker
Packs/FeedCloudflare
Packs/FeedSpamhaus
Packs/FeedDShield
```

File: Packs/ApiModules/ReleaseNotes/2_2_0.md (hunk 1 above)
Comment (dansterenson) [reply Packs/ApiModules/ReleaseNotes/2_2_0.md]: Done

**Review by yaakovi — CHANGES_REQUESTED** [review]
move the RN to a new file (bump each pack version by a revision).

you can use the `update-release-notes` command from the SDK to generate the RN.

File: Packs/FeedBlocklist_de/ReleaseNotes/1_0_6.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Blocklist_de Feed
+- Fixed an issue where tags field was not shown.
```
Comment (roysagi) [inline Packs/FeedBlocklist_de/ReleaseNotes/1_0_6.md:4]: please provide more detail - not shown where?

File: Packs/FeedBruteForceBlocker/ReleaseNotes/1_0_6.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### BruteForceBlocker Feed
+- Fixed an issue where tags field was not shown.
```
Comment (bakatzir) [inline Packs/FeedBruteForceBlocker/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedBlocklist_de/ReleaseNotes/1_0_6.md (hunk 3 above)
Comment (bakatzir) [inline Packs/FeedBlocklist_de/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedCloudflare/ReleaseNotes/1_0_6.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Cloudflare Feed
+- Fixed an issue where tags field was not shown.
```
Comment (bakatzir) [inline Packs/FeedCloudflare/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedDShield/ReleaseNotes/1_0_6.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### DShield Feed
+- Fixed an issue where tags field was not shown.
```
Comment (bakatzir) [inline Packs/FeedDShield/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedFeodoTracker/ReleaseNotes/1_0_9.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Feodo Tracker Hashes Feed (Deprecated)
+- Fixed an issue where tags field was not shown.
```
Comment (bakatzir) [inline Packs/FeedFeodoTracker/ReleaseNotes/1_0_9.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedMalwareDomainList/ReleaseNotes/1_0_6.md [hunk 8]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Malware Domain List Active IPs Feed
+- Fixed an issue where tags field was not shown.
```
Comment (bakatzir) [inline Packs/FeedMalwareDomainList/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedPlainText/ReleaseNotes/1_0_7.md [hunk 9]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Plain Text Feed
+- Fixed an issue where tags field was not shown.
```
Comment (bakatzir) [inline Packs/FeedPlainText/ReleaseNotes/1_0_7.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedSpamhaus/ReleaseNotes/1_0_6.md [hunk 10]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Spamhaus Feed
+- Fixed an issue where tags field was not shown.
```
Comment (bakatzir) [inline Packs/FeedSpamhaus/ReleaseNotes/1_0_6.md:4]:
```suggestion
- Fixed an issue where Tags defined in the feed instance configuration was not associated as an indicator field.
```

File: Packs/FeedBlocklist_de/ReleaseNotes/1_0_6.md (hunk 3 above)
Comment (dansterenson) [reply Packs/FeedBlocklist_de/ReleaseNotes/1_0_6.md]: Done

**Review by reutshal — APPROVED** [review]

**Review by yaakovi — APPROVED** [review]

**Review by reutshal — APPROVED** [review]
